### ISSUE TITLE
Scala 2.13 support

### ISSUE THREAD

**Issue by 2m** [matsluni/aws-spi-akka-http#3]
The following dependency libraries are yet to be available for Scala 2.13:

```
"org.elasticmq" %% "elasticmq-server" % "0.14.1" % "test"
"io.findify"    %% "s3mock"           % "0.2.5"  % "test"

**Comment by gabfssilva** [comment]
I'll work on #1 (use localstack)  in order to remove these dependencies.

**Comment by matsluni** [comment]
Scala 2.13.0-M5 is available with c5690c0c55b4ff0857ca03b58c9414313392272e and v0.0.5 (soon available on maven central).
Support for 2.13.0-RC1 has dependency on https://github.com/testcontainers/testcontainers-scala

**Comment by 2m** [comment]
Thanks a lot! Updated Alpakka connectors: https://github.com/akka/alpakka/pull/1641
